jupyterhub/app.py

```diff
@@ -2104,43 +2119,57 @@ async def init_role_assignment(self):
             # tokens need to be checked for permissions
             for kind in kinds:
                 orm_role_bearers = []
-                if kind in predef_role.keys():
-                    for bname in predef_role[kind]:
+                if kind in role_spec:
+                    for name in role_spec[kind]:
                         if kind == 'users':
```

Comment:
According to the [docs](https://jupyterhub.readthedocs.io/en/stable/rbac/roles.html#roles), the default role name is `user` not `users`. Is this another bug?